.env.example

```diff
@@ -13,9 +13,9 @@ ADMIN_JWT_SECRET=6YfHrcOhoJR9P/AUFpUMWw==
 JWT_SECRET=7Q+xaEnWZA1hikIx9e/fFg==
 API_TOKEN_SALT=jvdnndiwppccgrplclknsrgsvmevcfpqidywnaqnniacbcrsovsqohyyqnouagib
 APP_KEYS="toBeModified1,toBeModified2"
-TRANSFER_TOKEN_REMOTE=REPLACEME
+STRAPI_TRANSFER_TOKEN=REPLACEME
+STRAPI_TRANSFER_URL=https://REPLACEME/admin
 TRANSFER_TOKEN_LOCAL=REPLACEME
```

Comment:
`TRANSFER_TOKEN_LOCAL`isn't needed anymore then?

Reply:
ah right - removed this part from the readme today but forgot it here - will delete it now